templates/zerver/integrations/git.md

```diff
@@ -1,31 +1,57 @@
-Get Zulip notifications for your Git repositories!
+# Zulip Git integration
+
+Get Zulip notifications for pushes to your Git repositories!
+
+!!! tip ""
+
+    Zulip offers integrations for
+    [various repository hosting services][other-integrations], and also
+    supports this direct Git integration for unsupported services.
+
+!!! warn ""
+
+    This integration is meant to be run on the server-side.
```

Comment:
I assume this means that the integration needs to be run on the Zulip server? If so, then it would need to be set up by a server administrator, correct? If so, then I think that should be more clearly stated.